implementers.md

```diff
@@ -0,0 +1,10 @@
+# OCI Artifacts Implementations
+
+Projects or companies currently engaged in efforts to support OCI Artifacts
+
+* [Azure Container Registry](https://aka.ms/acr/artifacts)
+* [distribution/distribution](https://github.com/distribution/distribution/) - the reference implementation of the [OCI Distribution Spec][distribution-spec]
```

Comment:
we should drop "the reference implementation"

Reply:
@jdolitsky done :+1: